content/docs/user-guide/components/reference/gameplay/tag.md

```diff
@@ -1,151 +1,55 @@
 ---
-description: ' Use the Tag component to apply labels to entities in Open 3D Engine. '
-title: Tag
+linkTitle: Tag
+title: Tag Component
+description: Use the Tag component to apply labels to entities in Open 3D Engine (O3DE).
 ---
 
+Use the **Tag** component to apply one or more labels, or *tags*, to an entity. Use these tags to find or filter entities with particular labels.
 
+## Provider
 
-Use the **Tag** component to apply one or more labels, or tags, to an entity such as **burning** or **player**. You can use these tags to find or filter entities with particular traits. For example, you can set a weapon to inflict double damage to entities tagged as **burning**.
+[O3DE Core (LmbrCentral) Gem](/docs/user-guide/gems/reference/o3de-core)
 
-## EBuses - Request Bus Interface: TagGlobalRequestBus 
+## Tag properties 
 
-Use the following request function with the **TagGlobalRequestBus** EBus interface to communicate with other components of your game.
+![Tag component properties](/images/user-guide/components/reference/gameplay/tag-component.png)
 
-For more information about using the Event Bus (EBus) interface, see [Working with the Event Bus (EBus) system](/docs/user-guide/engine/ebus/).
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Tags** | An array of tags that are added to the entity. | EntityId | None | 
 
-### RequestTaggedEntities 
+## TagHelper
 
-Handlers respond if they have the tag (listening on the tag's channel). Use `AZ::EbusAggregateResults` to handle more than the first responder.
+| Request Name | Description | Parameter | Return | Scriptable |
+|-|-|-|-|-|
+| `GetEntitiesbyTag` | Returns the array of entities with a specific tag. | Tag: Crc32 | Array: EntityIds | Yes |
```

Comment:
It has been the norm (from early LY days) to pre-compute the CRC32 from strings, even in the scripting solutions.  I agree that is isn't as user-friendly as it could be, in contrast to components that have tag fields, they accept strings.  I will double check the methods, the tag bus has a few methods that have inaccurate names, I'll call those out here.

Reply:
Yep, looks to me like it's constructing a Crc32 from a string. I think it's appropriate to include both the alias and the type, as you've done here.

Reply:
`FindTaggedEntities` is the underlying method for `GetEntitiesbyTag`.  I'm not sure why there is a namechange, but TagHelper.GetEntitiesbyTag(tag) is the correct call.